apis/v1beta1/config_types.go

```diff
@@ -80,8 +82,70 @@ type Roles struct {
 	Custom     []NodeRole `json:"custom,omitempty"`
 }
 
+// Chart single helm addon
+type Chart struct {
+	Name      string `json:"name,omitempty"`
+	ChartName string `json:"chartname,omitempty"`
+	Version   string `json:"version,omitempty"`
+	// +kubebuilder:validation:Optional
+	Values   string `json:"values,omitempty"`
+	TargetNS string `json:"namespace,omitempty"`
+	// Timeout specifies the timeout for how long to wait for the chart installation to finish.
+	// +kubebuilder:validation:Optional
+	Timeout time.Duration `json:"timeout,omitempty"`
```

Comment:
Should we use the updated duration here like k0s? Or maybe just leave it out entirely?